src/main/java/mate/academy/model/Book.java

```diff
@@ -0,0 +1,46 @@
+package mate.academy.model;
+
+import java.math.BigDecimal;
+
+public class Book {
+    private Long id;
+    private String title;
+    private BigDecimal price;
+
+    public Book(String title, BigDecimal price) {
+        this.title = title;
+        this.price = price;
```

Comment:
It's not consistent to use both a constructor and setters for initializing an object. Choose either to set all fields via the constructor or to use the default constructor and setters.